gpu-operator/release-notes.rst

```diff
@@ -33,6 +33,42 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
 
 ----
 
+.. _v25.3.1:
+
+25.3.1
+======
+
+.. _v25.3.1-new-features:
+
+New Features
+------------
+
+* Added support for the following software component versions:
+
+  - NVIDIA Container Toolkit version v1.17.8
+  - NVIDIA Kubernetes Device Plugin v0.17.2
+  - Node Feature Discovery v0.17.3
+  - NVIDIA GDRCopy Driver v2.5.0
+
+* Added support for the following NVIDIA Data Center GPU Driver versions:
+
+  - 570.133.20 (recommended)
+  - 550.163.01
+  - 535.247.01
+
+* Added support for setting DCGM Exporter internalTrafficPolicy.
```

Comment:
@tariq1890 added a note here for https://github.com/NVIDIA/dcgm-exporter/pull/494, is this correct? is there anything else we should add about this? Is this something we should call out here https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#common-chart-customization-options?

Reply:
Yes, we can add a new entry to the chart customisation options

```
  dcgmExporter.service.internalTrafficPolicy: ["Cluster", "Local"] (default: Cluster)
```